### DISCUSSION TITLE
2012/07/09/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E8%A1%A5%E5%85%85%E5%86%85%E5%AE%B9%E5%92%8C%E6%B3%A8%E9%87%8A-%E7%AC%AC2%E7%89%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#356] [Announcements]
# 2012/07/09/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E8%A1%A5%E5%85%85%E5%86%85%E5%AE%B9%E5%92%8C%E6%B3%A8%E9%87%8A-%E7%AC%AC2%E7%89%88

营养药和美容护肤品

https://blog.qizhen.xyz/2012/07/09/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E8%A1%A5%E5%85%85%E5%86%85%E5%AE%B9%E5%92%8C%E6%B3%A8%E9%87%8A-%E7%AC%AC2%E7%89%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

高炜
2020年10月28日 21:16 编辑
阮老师，我是国内西安光机所一名工程师，我有个问题想请教您，问了好多人没人知道，所以想问问您，能给我留个邮箱或其他联系方式吗？谢谢啦。

回复
